_source/_posts/2020-03-09-what-i-learned-by-working-remotely-net.md

```diff
@@ -0,0 +1,75 @@
+
+## Working Remote is Not Easy For Everyone
+
+Isolation can creep up on people if they don't have the physical presence of their peers. Part of how I handled the sudden shift from working in a room with a team to completely alone was to utilize white noise. I'd put on a podcast, listen to music or play a movie I've seen before on a television somewhere in the room. Eventually, I graduated to using just Spotify playlists. Trust me, I understand that this transition can be difficult especially if you are a people person. While I don't always feel the need to be around my team, there were most definitely times I wish I had been. In those moments, I would also reach out to my developer friend network outside of my company and that helped a ton. Sometimes you miss the conversations that happen purely because you happened to be passing by someone in a hallway. Part of how I handled that was getting involved in local developer user groups, attending conferences or participating in hackathons to get my community fix. I also found local developer friends that I would regularly schedule lunch with to get both of us away from work for a bit. The biggest takeaway I had was being *intentional* in contacting my team on a regular basis. If it had been a minute since I had chatted with them one on one, I would just start a direct message asking how they've been. It takes time and effort to begin a conversation for interpersonal reasons, and I always appreciated when a coworker would check up on me, or share their latest tech purchase.
+
+## Working Remotely Saved Me from Burnout
```

Comment:
http://www.conissaunce.com/pages/infoq-title-case.html says "from" should be capitalized.